simulator/travis/minimesosFile

```diff
@@ -132,18 +132,18 @@ minimesos {
 
             ports {
                 role = "*"
-                value = "[31000-32000]"
+                value = "[31400-31500]"
             }
             ports {
-                role = "*"
-                value = "[32000-33000]"
+                role = "cook"
```

Comment:
Please leave at least one of these as "*" as I want to make sure nothing weird happens with roles.